docs/tutorial.md

```diff
@@ -0,0 +1,285 @@
+<!--
+title: Tutorial: Your First Serverless Framework Project
+menuText: Your First Project
+-->
+
+# Tutorial: Your First Serverless Framework Project
+
+<iframe width="700" height="394" src="https://www.youtube.com/embed/dMVfqCTzuwk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+This guide helps you create and deploy a REST API with Serverless Framework and AWS.
```

Comment:
Let's not confuse users with `REST` where we're using `httpApi` event type - maybe it should say HTTP API instead of REST API?